compiler/ballerina-lang/src/main/java/io/ballerina/projects/DeadFunctionVisitor.java

```diff
@@ -0,0 +1,48 @@
+/*
+ *  Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 *  Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
```

Reply:
Check other places as well

Reply:
Need to remove the "All Rights Reserved" part as well.

Reply:
Will change to this,
`/*
 *  Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
 *
 *  WSO2 LLC. licenses this file to you under the Apache License,
 *  Version 2.0 (the "License"); you may not use this file except
 *  in compliance with the License.
 *  You may obtain a copy of the License at
 *
 *    http://www.apache.org/licenses/LICENSE-2.0
 *
 *  Unless required by applicable law or agreed to in writing,
 *  software distributed under the License is distributed on an
 *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 *  KIND, either express or implied. See the License for the
 *  specific language governing permissions and limitations
 *  under the License.
 */`

Reply:
Updated in commit 70012d7a0af8c168e0ffe63d303385078e4caa8d